views/js/pciCreator/ims/mathEntryInteraction/creator/widget/states/Question.js

```diff
@@ -26,7 +26,7 @@ define([
     'taoQtiItem/qtiCreator/editor/containerEditor',
     'tpl!mathEntryInteraction/creator/tpl/propertiesForm',
     'tpl!mathEntryInteraction/creator/tpl/addGapBtn',
-    'mathEntryInteraction/helper/mathInPrompt'
+    'mathEntryInteraction/creator/helper/mathInPrompt'
```

Comment:
You removed the `mathInPrompt` calls from the runtime, but it looks like you continue to use it in the creator. Is this because of the rendering fix it brings there? Wouldn't the approach be likely to change in the next part of the fix?

Because from a QA perspective, it could be safer if the v2.0.2 of this PCI is a full revert to the exact v2.0.0 code, before the #312 changes. But if you want to leave the new code in, I'll trust your judgement, your knowledge is deeper than mine.

Reply:
Correct. I kept it for having the math element properly rendered in authoring.
I still don't know if this is needed with the new approach, or if it will be replaced. I found it safer to keep it for now.